Gemfile

```diff
@@ -18,6 +18,7 @@ gem 'redis'
 gem 'database_cleaner'
 
 gem 'pry'
+gem 'pry-nav'
```

Comment:
Do we need this?

Reply:
It helped with navigating the debugger. I can remove it if you like.

Reply:
Yeah, let's remove it. Please ❤️